keyless/circuit-data/templates/mainTemplate.circom

```diff
@@ -194,7 +197,8 @@ template identity(
     signal ev_fail <== AND()(uid_is_email, not_ev_in_jwt);
     ev_fail === 0;
 
-    JWTFieldCheck(maxEVKVPairLen, maxEVNameLen, maxEVValueLen)(ev_field, ev_field_len, ev_index, ev_name_len, ev_value_index, ev_value_len, ev_colon_index, ev_name, ev_value);
+    log("email_verified");
+    ParseJWTFieldWithUnquotedValue(maxEVKVPairLen, maxEVNameLen, maxEVValueLen)(ev_field, ev_name, ev_value, ev_field_len, ev_name_len, ev_value_index, ev_value_len, ev_colon_index);
```

Comment:
There appear to be some providers as of a few years ago that incorrectly use strings for email_verified: https://community.auth0.com/t/userinfo-email-verified-field-string-boolean-or-both/27553

Reply:
This might actually be fine, since `ParseJWTFieldWithUnquotedValue` seems to allow for quoted values as well